### ISSUE TITLE
Cannot create files on Redmi Note 8T

### ISSUE THREAD

**Issue by cyberdo** [zeapo/Android-Password-Store#1150]
First off: I can create regular folders
Second: I can not create password files, the UI just blinks and I get back to the password edit-screen
Third: I cannot create folders with custom GPG key IDs. I get the following crash report:

```
java.lang.IllegalStateException: Fragment FolderCreationDialogFragment{9c883a9} (d2652d5d-9d37-4ba0-8e6f-5743514cce73)} is attempting to registerForActivityResult after being created. Fragments must call registerForActivityResult() before they are created (i.e. initialization, onAttach(), or onCreate()).
	at androidx.fragment.app.Fragment.prepareCallInternal(Fragment.java:5)
	at androidx.fragment.app.Fragment.registerForActivityResult(Fragment.java:1)
	at com.zeapo.pwdstore.ui.dialogs.FolderCreationDialogFragment.access$createDirectory(FolderCreationDialogFragment.kt:14)
	at -$$LambdaGroup$js$-EhQtwzYX4RN32t-7spKSIuifK8.onClick(java-style lambda group:34)
	at android.view.View.performClick(View.java:6608)
	at com.google.android.material.button.MaterialButton.performClick(MaterialButton.java:2)
	at android.view.View.performClickInternal(View.java:6585)
	at android.view.View.access$3100(View.java:785)
	at android.view.View$PerformClick.run(View.java:25921)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:201)
	at android.app.ActivityThread.main(ActivityThread.java:6864)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
```

I haven't connected the phone to a computer for reading the log. Is there an app that can do it or do I need the Android SDK?

**Comment by msfjarvis** [comment]
As mentioned in the issue template, bug reports that don't fill the template correctly will be immediately closed.

**Comment by cyberdo** [comment]
Good to know. Where can find this template? I found nothing on Play Store, nor here on Github.

**Comment by msfjarvis** [comment]
> Good to know. Where can find this template? I found nothing on Play Store, nor here on Github.

Filing a new issue is supposed to offer you the option. I've gone ahead and backported the fix for it regardless (#1151), and v1.12.1 will be released shortly.

**Comment by cyberdo** [comment]
Thank you!

(I see nothing of the kind but I will at least try to imitate another issue better next time)
